src/connectionbenchmark/connectionbenchmark.cpp

```diff
@@ -18,7 +18,8 @@ Logger logger(LOG_MODEL, "ConnectionBenchmark");
 }
 
 ConnectionBenchmark::ConnectionBenchmark()
-    : m_downloadUrl(Constants::BENCHMARK_DOWNLOAD_URL) {
+    : m_downloadUrl(Constants::BENCHMARK_DOWNLOAD_URL),
```

Comment:
Why do you need these 2 var members? Can you use `Constants` when needed?

Reply:
Setting them now with `setUploadUrl()` in the `start()` method.